docs/visualization/helper_api/regex_expressions_get_callbacks.md

```diff
@@ -0,0 +1,16 @@
+# Regex expressions of get_callbacks()
```

Comment:
I think regex and unix filename pattern maching are different
How abount
`Unix shell-style wildcards for get_callbacks` ?

https://docs.python.org/3/library/fnmatch.html

Reply:
Thank you. I misunderstood.
I changed to `Wildcards for get_callbacks`.